dvc/config.py

```diff
@@ -320,8 +330,10 @@ def rel(path):
                 return path
 
             if isinstance(path, RelPath) or not os.path.isabs(path):
-                return relpath(path, conf_dir)
-            return path
+                path = relpath(path, conf_dir)
+
+            posix_path = str(PurePosixPath(Path(path)))
```

Comment:
So looks like this will also work with windows abs paths, nice! Could you please add a unit test or something for this helper? Just to make sure it behaves as expected. 

Btw, maybe our `PathInfo.as_posix` from `dvc/path_info.py` might come in handy.

Reply:
Ah nice, I've changed to `PathInfo` for consistancy